.github/workflows/check.yml

```diff
@@ -2,8 +2,6 @@ name: check
 on:
   push:
   pull_request:
-  schedule:
```

Comment:
Any reason to remove these?

Reply:
I can back out the change, but I was wondering if that would fix the workflow being disabled :shrug:

Reply:
Let's make it part of another PR if we'd do it.